contracts/governance/README.adoc

```diff
@@ -70,14 +70,20 @@ NOTE: Functions of the `Governor` contract do not include access control. If you
 
 {{GovernorTimelockCompound}}
 
+{{IGovernorTimelock}}
```

Comment:
Should interfaces be added here. I'm not sure.

Reply:
other docs including this one have interfaces in them, if you want I can remove them

Reply:
right, in the lower part we do include the interfaces...

It should probably have gone in a separate PR, but if @frangio or @ernestognw approves that that is fine with me.

Reply:
I noticed this and it's not ideal but I'm fine with doing it in this PR.

Reply:
I think it's removed [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4358/commits/bd74728f5630baf36622c4782d08f8c8b31d1f69) so not much to add other than saying I agree the interface is not needed.